apps/style/applab/style.scss

```diff
@@ -206,6 +206,7 @@ div#visualizationResizeBar {
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
+  cursor: crosshair;
```

Comment:
Yeah, let's remove it from this PR. It might seem like a bug without the rest of the feature.

Thanks for continuing to work on this debug console stuff!  It's so cool!